modules/geocoder_field/src/Plugin/Geocoder/Preprocessor/File.php

```diff
@@ -42,6 +42,9 @@ class File extends PreprocessorBase {
    *   The plugin_id for the plugin instance.
    * @param mixed $plugin_definition
    *   The plugin implementation definition.
+   * @param FileSystemInterface $file_system
+   *   The file system object.
+   *
    * @para, \Drupal\Core\File\FileSystemInterface $file_system
```

Comment:
It was a typo '@para,' instead '@param'. But the rest is better as it was because the type hint should be a fully qualified class name. Also the explanation is better because it's a service.